.github/workflows/commit-message-check.yml

```diff
@@ -9,5 +9,5 @@ jobs:
       - name: Check Commit Type
         uses: gsactions/commit-message-checker@v1
         with:
-          pattern: '(^Bump[a-zA-Z0-9 ./@_-]+)|(CANVAS-[0-9]+\ -\ .{5,})|(^Revert[a-zA-Z0-9 ./@_-]+)'
```

Comment:
@Auroro0804 Keep this change. It allows reverts to not fail the commit message check action and shouldn't interfere  with app functionality

Reply:
@rlorenzo I added it back. You may see https://github.com/ucla/lti-media-resources/blob/master/.github/workflows/commit-message-check.yml#L12.